packages/bot-web-ui/src/stores/run-panel-store.js

```diff
@@ -141,6 +144,10 @@ export default class RunPanelStore {
         );
     }
 
+    setShowBotStopMessage(value) {
```

Comment:
as a question, is it possible to add this to a custom hook instead of the store?
the reason is that as you know we are trying to reduce the usage of the stores, not sure if this can be out or not.